Sources/BraintreeVenmo/BTVenmoClient.swift

```diff
@@ -83,21 +83,23 @@ import BraintreeCore
         apiClient.sendAnalyticsEvent(BTVenmoAnalytics.tokenizeStarted, isVaultRequest: shouldVault, linkType: linkType)
         let returnURLScheme = BTAppContextSwitcher.sharedInstance._returnURLScheme
 
-        if returnURLScheme.isEmpty {
-            NSLog(
-                "%@ Venmo requires a return URL scheme to be configured via [BTAppContextSwitcher setReturnURLScheme:]",
-                BTLogLevelDescription.string(for: .critical)
-            )
-            notifyFailure(with: BTVenmoError.appNotAvailable, completion: completion)
```

Comment:
Also, I feel like `VenmoError.appNotAvailable`, which reads `The Venmo app is not installed on this device, or it is not configured or available for app switch.` is not the right error here. I think it should instead be `VenmoError.invalidReturnURL`

Reply:
IMO we can treat that as a bug and fix it in this PR (line 92 below)

Reply:
Yep, I mentioned that [here](https://github.com/braintree/braintree_ios/pull/1503#discussion_r1924376663), will update this to a more appropriate error.

Reply:
Updated: https://github.com/braintree/braintree_ios/pull/1503/commits/4f1db1f69af23678ee79b6a591568478f556710d